app/controllers/annotation_categories_controller.rb

```diff
@@ -204,6 +204,11 @@ def upload
     redirect_to assignment_annotation_categories_path(assignment_id: @assignment.id)
   end
 
+  def annotation_text_stats
```

Comment:
Based on the data that's actually being returned, `annotation_text_uses` is probably a better name for the route. In fact, you should probably change "stats" -> "uses" throughout.